pyproject.toml

```
@@ -1,12 +1,115 @@
 [build-system]
 requires = [
-	"wheel", "setuptools",
+	"setuptools",
```

```suggestion
	"setuptools >= 61.0.0",
```

for `pyproject.toml` configuration.